src/test/lib/import_export/csv/csv_parser_test.cpp

```diff
@@ -332,7 +330,7 @@ TEST_F(CsvParserTest, WithScheduler) {
 }
 
 TEST_F(CsvParserTest, ImmutableChunks) {
-  const auto csv_file = "resources/test_data/csv/float_int_large.csv"s;
+  const auto csv_file = std::string{"resources/test_data/csv/float_int_large.csv"};
```

Comment:
I only skimmed over the files, but do we test cases where the table already exists?

Reply:
And where, e.g., the table already stores data?

Reply:
This would be the wrong file to put it, but its also missing in `import_test.cpp`. I will add one.

Reply:
Do we check that the assertation is raised when we have a json file but also the table exists?

Reply:
I added a test for that as well.